src/input.js

```diff
@@ -11,40 +11,71 @@ export const mydata = () => {
     if (e.key === 'Enter') {
       const totaltasks = create();
       const task = {
-        check: true,
-        des: document.querySelector('.main').value,
+        description: document.querySelector('.main').value,
+        completed: false,
         index: totaltasks.length,
       };
       totaltasks.push(task);
-      localStorage.setItem('todo', JSON.stringify(totaltasks));
+      localStorage.setItem('lists', JSON.stringify(totaltasks));
     }
   });
 };
 
-const totaltasks = [
-  {
-    description: 'clean the room',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'do homework',
-    completed: false,
-    index: 2,
-  },
-];
-
 export const display = () => {
   const inner = document.querySelector('.show');
-  // const totaltasks = create();
+  const totaltasks = create();
   for (let i = 0; i < totaltasks.length; i += 1) {
     inner.innerHTML += `
-       <div class="container"> 
+       <div class="container" id='con${i}'> 
        <div>
        <input class='checkbox' type="checkbox" ${totaltasks.completed ? 'checked' : ' '}/>
-       ${totaltasks[i].description}</div>
-       <i class="fa-solid fa-ellipsis-vertical"></i>
+      <span id ='iddes' class="des${i}"> 
+      <input id='in${i}' readonly type="text" value="${totaltasks[i].description}"></input>
+      </span></div>
+     <span id="icon${i}" > <i class="fa-solid fa-ellipsis-vertical" id="${i}"></i></span>
        </div>
         `;
   }
 };
```

Comment:
- Please ensure that the index is set to the value of the new array length (i.e. if a 5th task is added to the list, that task's index should equal 5). At the moment I have 6 tasks and I'm getting an index of 5 instead of 6 👍🏽 
![Screenshot from 2023-07-21 16-21-10](https://github.com/MayPyone/to-do-list/assets/44624138/fef4ea5c-3754-45d5-8ed7-f31b5ff342b3)